enterprise/server/remote_execution/workspace/workspace.go

```diff
@@ -91,6 +92,15 @@ func (ws *Workspace) SetTask(task *repb.ExecutionTask) {
 	ws.dirHelper = dirtools.NewDirHelper(ws.Path(), cmd.GetOutputFiles(), cmd.GetOutputDirectories())
 }
 
+func (ws *Workspace) SetOwner(uid, gid int) error {
```

Comment:
Yeah, I guess since it doesn't support numeric user IDs, it's probably even more inconvenient to ask users to update their `USER` to a numeric ID (which requires building & pushing a new image), than it would be to ask them to tack on an extra platform prop.

I'd be fine with doing this as a platform prop for now and then maybe later switching to a more automated approach if we think it's worth it.

Reply:
Yeah if we don't get non-numeric user ID support from docker, I agree that a platform arg is gonna be easier and less code here, rather than having partial support only.

Reply:
Done